docs/sources/reference/commandline/cli.md

```diff
@@ -1100,6 +1100,10 @@ and Docker images will report:
 
     untag, delete
 
+The `--since` and `--until` parameters can be Unix timestamps, RFC3339
+dates or Go duration strings (e.g. `10m`, `1h30m`) computed relative to
```

Comment:
@thaJeztah I don't think that's needed. the feature is useful as is, no need for restricting. we support nanoseconds as well in rfc3339 format but we slightly omit that data and have seconds precision.